File: crates/spur-graph/tests/fixtures/rename_corpus/rust/adversarial_01/old.rs
pub fn rust_full_rewrite_before(alpha_count: i32, beta_limit: i32) -> i32 {
    let mut total = alpha_count;
    for step in 0..beta_limit {
        total += step * 3;
        if total % 2 == 0 {
            total -= 1;
        }
    }
    total
}

File: crates/spur-graph/tests/fixtures/rename_corpus/typescript/adversarial_01/old.ts
export function tsFullRewriteBefore(alphaCount: number, betaLimit: number): number {
  let total = alphaCount;
  for (let step = 0; step < betaLimit; step += 1) {
    total += step * 3;
    if (total % 2 === 0) {
      total -= 1;
    }
  }
  return total;
}
